doc/book/admin/monitoring/getting_started.rst

```diff
@@ -0,0 +1,105 @@
+..  _monitoring-getting_started:
+
+Getting started with monitoring
+===============================
+
+Example on GitHub: `sharded_cluster_crud_metrics <https://github.com/tarantool/doc/tree/latest/doc/code_snippets/snippets/sharding/instances.enabled/sharded_cluster_crud_metrics>`_
+
+Tarantool allows you to configure and expose its :ref:`metrics <metrics-reference>` using a :ref:`YAML configuration <configuration>`.
+You can also use the :ref:`metrics <metrics-api_reference>` module to create and collect custom metrics.
+
+
+
+
+.. _monitoring_configuring_metrics:
+
+Configuring metrics
+-------------------
+
+To configure metrics, use the :ref:`metrics <configuration_reference_metrics>` section in a cluster configuration.
+The configuration below enables all metrics, excluding :ref:`Vinyl <metrics-reference-vinyl>`-specific ones:
```

Comment:
```suggestion
The configuration below enables all metrics excluding :ref:`vinyl <metrics-reference-vinyl>`-specific ones:
```
`vynil` seems to be all-lowercase: https://docs.d.tarantool.io/en/doc/metrics/concepts/engines/vinyl/

Reply:
Also, `excluding.. ` is a restrictive clause, so no comma is needed.